### PR TITLE
Add url for search providers in the UI. [SDESK-4905]

### PR DESCRIPTION
**Pull request by GyanP**
depends on https://github.com/superdesk/superdesk-client-core/pull/3266

### CODE REVIEW

**Comment by ride90** [comment]
depends on https://github.com/superdesk/superdesk-core/pull/1762

File: server/anp/search_providers/talpa_video.py [hunk 1]
```diff
@@ -42,6 +41,7 @@ class TalpaVideoSearchProvider(superdesk.SearchProvider):
 
     def __init__(self, provider):
         super().__init__(provider)
+        self.url = provider.get('config', {}).get('url', '')
```
Comment (petrjasek) [inline server/anp/search_providers/talpa_video.py:44]: just wondering how many urls are available here, maybe an env variable would be enough for it

Comment (GyanP) [reply server/anp/search_providers/talpa_video.py:44]: Yes, that could be an option, but the use case was to make it possible to change the URL from the UI as mentioned in the ticket.

Comment (petrjasek) [inline server/anp/search_providers/talpa_video.py:44]: would be good to do some `assert self.url` after assigning

**Review by petrjasek — APPROVED** [review]

**Pull request merged**